app/controllers/index.rb

```diff
@@ -0,0 +1,57 @@
+# require 'sinatra'
+
+get '/' do
+  @sealions = Sealion.all
+  erb :index
+end
+
+get '/sealions' do
+  @sealions = Sealion.all
+  erb :index
+end
+
+get '/sealions/new' do
+  erb :new
+end
+
+post '/sealions/new' do
+  new_sealion = Sealion.create(first_name: params[:first_name], last_name: params[:last_name], age: params[:age], location: params[:location], quirk: params[:quirk])
+  redirect "/sealions/#{new_sealion.id}"
+end
+
+get '/sealions/:id' do
+  @sealion = Sealion.find_by(id: params[:id])
+  erb :show
+end
+
+get '/sealions/:id/edit' do
+   @sealion = Sealion.find_by(id: params[:id])
+   erb :edit
+ end
+
+put '/sealions/:id/edit' do
+  @sealion = Sealion.find_by(id: params[:id])
+
+  if @sealion
```

Comment:
You could invert the sense of this: `return 404 unless @sealion`. Then you wouldn't have to indent.